src/main/java/org/folio/consortia/service/impl/UserTenantServiceImpl.java

```diff
@@ -294,6 +296,20 @@ private FolioExecutionContext prepareContextForTenant(String tenantId, FolioExec
     return new DefaultFolioExecutionContext(folioModuleMetadata, context.getOkapiHeaders());
   }
 
+  public void deleteShadowUsers() {
+    FolioExecutionContext currentTenantContext = (FolioExecutionContext) folioExecutionContext.getInstance();
+    List<UserTenantEntity> userTenantEntities = userTenantRepository.getByUserIdAndIsPrimaryFalse().orElse(null);
+    if (Objects.nonNull(userTenantEntities)) {
```

Comment:
after removing optional we can use CollectionUtils.isNotEmpty method to simplify things

Reply:
fixed